aks/terraform/modules/cluster/main.tf

```diff
@@ -74,6 +77,10 @@ resource "azurerm_kubernetes_cluster" "cluster" {
     vnet_subnet_id  = var.subnet_id
     max_pods        = local.worker_node_max_pods
     zones           = local.availability_zones
+
+    upgrade_settings {
```

Comment:
This is unrelated to tagging - added because a second 'terraform apply' show this as a delta and remove it.